### DISCUSSION TITLE
Users assuming "Download ZIP" points to build artifacts

### DISCUSSION THREAD

**Discussion by Sauceke** [community/community#5348] [Other Feature Feedback, Questions, & Ideas]
Workflow I see often (happens on a weekly basis in my community):
User (layperson) wants to download a program. Misses the Releases section completely and goes for the most eye-catching thing on the page, a big green button. Sees "Download ZIP". Downloads the ZIP. The ZIP doesn't have what user is looking for, only a bunch of "weird files". User seeks help on a forum/chat, which shortly arrives in the form of a screenshot with the Releases section circled in red. User can now proceed to download the program.

How I think this workflow could be simplified:
If a repository has any release artifacts, there should be a dedicated Download button, e. g. next to the Code button, which points to the latest release page. I think this would save time for both end users and volunteers trying to help them.

Side note: even after navigating to the release page, some users will mistakenly download the zipped source code automatically included with the release artifacts. This seems to happen in only a small minority of cases, but I think it's worth mentioning anyway.

**Comment by feldon30** [comment]
Releases should be SO much more prominent then they are. They're barely visible on the page.
